src/workers/block_listenser.rs

```diff
@@ -55,10 +55,6 @@ lazy_static::lazy_static! {
         "Time to receive finalized block from block subscribe",
     ))
     .unwrap();
-    static ref FIN_BLOCKS_RECV: IntCounter =
```

Comment:
why do you want to remove this? Could be interesting later to debug.

Reply:
The `TT_RECV_FIN_BLOCK` and `TT_RECV_FIN_BLOCK` already has a count property in them.
So this is the same thing done twice here